File: backend/components/metiers/threed/tests_centralises/legacy/samples/compliance/test_compliance_sample.js
// Test ultra avancé pour compliance_sample.js
const complianceSample = require('../../../legacy/samples/compliance/compliance_sample');

describe('Compliance Sample', () => {
  it('doit valider la conformité RGPD', () => {
    expect(complianceSample({consent: true})).toBe(true);
  });
  it('doit refuser un objet sans consent', () => {
    expect(complianceSample({})).toBe(false);
  });
  // ...autres tests avancés, edge cases, conformité...
});

File: backend/components/metiers/threed/tests_centralises/legacy/samples/compliance/test_compliance_sample.py
"""
Test ultra avancé pour compliance_sample.py
"""

from threed.legacy.samples.compliance.compliance_sample import (
    compliance_sample,
)


def test_compliance_consent():
    assert compliance_sample({"consent": True}) is True


def test_compliance_no_consent():
    assert compliance_sample({}) is False


# ...autres tests avancés, edge cases, conformité...
